replication-api/src/main/java/org/codice/ditto/replication/api/QueryService.java

```diff
@@ -0,0 +1,9 @@
+package org.codice.ditto.replication.api;
+
+import java.util.List;
+import org.codice.ditto.replication.api.data.Query;
+
+public interface QueryService {
+
+  public List<Query> getQueries();
```

Comment:
❗️ Javadoc
❓ Could we return a `Stream` instead of a collection?

Reply:
yes.